src/twisted/words/protocols/irc.py

```diff
@@ -2898,7 +2908,7 @@ class DccChat(basic.LineReceiver, styles.Ephemeral):
     delimiter = CR + NL
     client = None
     remoteParty = None
-    buffer = ""
+    buffer = b""
```

Comment:
If this is `bytes`, then `DccChat.lineReceived` should decode the line before sending it to `self.client.privmsg`, or `IRCClient` should do all of its work in `bytes` and rely on a higher level API to handle decoding.